maps/LastStand.MX2.lua

```diff
@@ -445,23 +447,23 @@ end
 
 function OnMapVictory()
 	MessageBox("Congratulations!");
-	MessageBox("Lord Varuun looks at you in despise, shouting \"you may have won this battle, but you will never win the war - we will meet again!\". He raises his hands and a cloud of smoke appears around him. When it clears out, he is no longer there.");
+	MessageBox("Lord Varuun glares despisingly apon you, shouting \"you may have won this battle, but you will never win the war - we will meet again!\". He raises his hands and a cloud of smoke appears around him. When it clears out, he is no longer there.");
```

Comment:
http://www.differencebetween.net/language/words-language/difference-between-upon-and-apon/

This would have been fine unchanged. Change "apon" to "upon," and this is good to go.